.travis.yml

```diff
@@ -13,7 +13,6 @@ before_install:
 
 install:
   - npm install
-  - npm run vscode:prepublish
 
 script:
   - npm test --silent
```

Comment:
Maybe we should remove this script section as well. That way, at least, we can see what the error is all about? I'm really not sure why this `--silent` flag was provided in the first place!